src/phoneInput.tsx

```diff
@@ -103,7 +115,7 @@ export class PhoneInput extends React.Component<PhoneInputProps> {
 
   handleSearch = (event: any) => {
```

Comment:
@DanielFuchs2  change to `handleSearch: React.ChangeEventHandler<HTMLInputElement>`
remove "any"
Same for handleClick